weave-js/src/components/FancyPage/useProjectSidebar.ts

```diff
@@ -9,12 +9,16 @@ export const useProjectSidebar = (
   viewingRestricted: boolean,
   hasModelsData: boolean,
   hasWeaveData: boolean,
-  isLargeWorkspaceModeEnabled: boolean
+  isLargeWorkspaceModeEnabled: boolean,
+  hasTraceBackend: boolean = true
 ): FancyPageSidebarItem[] => {
-  const isNoData = !hasModelsData && !hasWeaveData;
-  const isModelsOnly = hasModelsData && !hasWeaveData;
-  const isWeaveOnly = !hasModelsData && hasWeaveData;
-  const isBothData = hasModelsData && hasWeaveData;
+  const showModelsData = hasModelsData || !hasTraceBackend;
```

Comment:
I think the names are really confusing here, maybe we can think of something better. Like with showModelsData - probably "SidebarItems" would be better than "Data", and it's kind of weird to have it be true but hasModelsData be false.